tests/semantic_analyser.cpp

```diff
@@ -5036,8 +5038,8 @@ TEST_F(SemanticAnalyserTest, variable_declarations)
   test("begin { let $a: struct x *; }");
   test("begin { let $a: struct task_struct *; $a = curtask; }");
   test("begin { let $a: struct Foo[10]; }");
-  test("begin { if (1) { let $x; } $x = 2; }");
-  test("begin { if (1) { let $x; } else { let $x; } let $x; }");
+  test("begin { if (pid) { let $x; } $x = 2; }");
```

Comment:
why did these need changing? because of branch pruning?

Reply:
Yes, I'm just using `pid` to keep both branches in.